src/dev/riscv/LupV.py

```diff
@@ -96,6 +103,11 @@ class LupV(Platform):
     # PLIC
     plic = Param.Plic(Plic(pio_addr=0xc000000), "PLIC")
 
+    # LUPIO RNG
+    lupio_rng = LupioRNG(pio_addr=0x20005000)
+    # Interrupt ID for PLIC
+    lupio_rng_int_id = Param.Int(0x3, "PLIC LupioRNG interrupt ID")
+
```

Comment:
I'm bugging on this code a little bit. Shouldn't the interrupt ID be a parameter to the RNG device upon creating it?

At the moment, it seems like the interrupt ID is internally hardcoded in the RNG device and hardcoded here, which means that if they don't match, the code likely breaks :/